src/main/java/org/apache/sling/models/jacksonexporter/impl/JacksonExporter.java

```diff
@@ -124,6 +120,9 @@ public boolean isSupported(@NotNull Class<?> clazz) {
         }
     }
 
+    @Reference(name = "moduleProvider", service = ModuleProvider.class,
+            cardinality = ReferenceCardinality.MULTIPLE, policy = ReferencePolicy.DYNAMIC,
+            bind = "bindModuleProvider", unbind = "unbindModuleProvider")
     protected void bindModuleProvider(final ModuleProvider moduleProvider, final Map<String, Object> props) {
```

Comment:
Let us rely on field injection here.

Reply:
to be handled in https://issues.apache.org/jira/browse/SLING-10964